Framework/Algorithms/src/SumSpectra.cpp

```diff
@@ -549,20 +556,18 @@ void SumSpectra::doFractionalSum(MatrixWorkspace_sptr outputWorkspace,
       for (size_t yIndex = 0; yIndex < m_yLength; ++yIndex) {
         const double yErrorsVal = YErrors[yIndex];
         if (std::isnormal(yErrorsVal)) { // is non-zero, nan, or infinity
-          const double errsq =
-              yErrorsVal * yErrorsVal * FracArea[yIndex] * FracArea[yIndex];
+          const double errsq = yErrorsVal * yErrorsVal;
           YErrorSum[yIndex] += errsq;
           Weight[yIndex] += 1. / errsq;
-          YSum[yIndex] += YValues[yIndex] * FracArea[yIndex] / errsq;
```

Comment:
Can I ask why this is not correct and how it differs from doing the unfinalize and simple sum?

Reply:
@Mantid-Matthew Previously SumSpectra did not check if the input was finalized so if it was already unfinalized the processing was not consistent. The unfinalize call leaves the data in a known state.

Reply:
Thanks for the explanation. I hadn't considered the case where it was unfinalized first. Thanks.

Reply:
Added a test to SumSpectraTest to cover this case.